src/alfasim_sdk/_internal/alfacase/case_to_alfacase.py

```diff
@@ -40,6 +40,8 @@ def format_list(values: List[Any], *, enable_flow_style: bool = False):
     """
     import ruamel
 
+    # TODO: arthur: Review this workaround before merging!!!
+    return values
     retval = ruamel.yaml.comments.CommentedSeq(values)
```

Comment:
I think this will work.
```suggestion
    import strictyaml

    if hasattr(strictyaml, 'ruamel'):
        ruamel = strictyaml.ruamel
    else:
        import ruamel

    retval = ruamel.yaml.comments.CommentedSeq(values)
```

Note that the original code will work with ruamel/stricyaml from pypi, it fail on alfasim because the package/version of strictyaml is vendoring ruamel.
This code uses the vendored ruamel version if it is being vendored and falls back to the original ruamel.

Probably a better solution is updating the package we are using on alfasim.

Reply:
I will add this to
https://github.com/ESSS/alfasim-sdk/pull/301